src/pages/public_registration/RegistrationLandingPage.tsx

```diff
@@ -47,52 +46,36 @@ export const RegistrationLandingPage = () => {
     registrationQuery.refetch();
   };
 
-  const backNavigationPath = window.location.pathname.startsWith(UrlPathTemplate.MyPageMyMessages)
-    ? UrlPathTemplate.MyPageMyMessages
-    : window.location.pathname.startsWith(UrlPathTemplate.Tasks)
-    ? UrlPathTemplate.Tasks
-    : '';
-
   return (
-    <section>
-      {backNavigationPath && (
-        // TODO: Redundant when NP-44809 is completed
-        <Link to={backNavigationPath}>
-          <Button startIcon={<ArrowBackIcon />} variant="outlined" sx={{ mb: '0.5rem', textTransform: 'none' }}>
-            {t('common.back')}
-          </Button>
-        </Link>
-      )}
+    <Box
+      component="section"
+      sx={{
+        display: 'grid',
+        gridTemplateColumns: '4fr 1fr',
```

Comment:
Har endret dette fra `5fr 1fr` til `4fr 1fr` for å få en bittelitt større høyremeny. Tipper det blir flere endringer rundt dette i tiden fremover